source/getting-started/configuration.markdown

```diff
@@ -9,85 +9,39 @@ sharing: true
- - Now Restart Home Assistant using the "restart" in the Server management section.
-
-### {% linkable_title Editing config via Samba/Windows Networking %}
-
-Maybe you are not a big fan of our web editor and want to use a text editor on your computer instead. This is possible by sharing the configuration over the network using the Samba add-on, which can be installed from the Hass.io add-on store.
-
-After you have installed it, click on START. Hass.io should now be available in the networking tab on your computer. Use a text editor like the free [Visual Studio Code](https://code.visualstudio.com/) to edit `configuration.yaml`.
-
-## {% linkable_title Configuring integrations %}
+ - Most changes in `configuration.yaml` require Home Assistant to be restarted to see the changes. You can verify that your changes are acceptable by running a config check. Do this by clicking on Configuration in the sidebar, click on General and click on the "Check Config" button. When it's valid, it will show the text "Configuration valid!".
+ - Now Restart Home Assistant using the "restart" in the Server management section on the same page.
 
-Now that you are able to edit the configuration, it's time to set up some of your devices and services. Each service and device will have its own instructions on how to be integrated. Find  your devices and services on the [components overview page](/components/).
-
-<p class='note'>YAML can be a little daunting at first. A lot is possible! [Here is some more info.](/docs/configuration/devices/)</p>
-
-For your first integration, you'll create a virtual sensor that generates a random integer value between 0 and 20 every 30 seconds.  
+<p class='img'>
+<img src='/images/screenshots/configuration-validation.png' />
+Screenshot of the "General" page in the configuration panel.
+</p>
 
-To create this random value sensor [random values](/components/sensor.random/), enter the following to the bottom of your `configuration.yaml` file, and restart Home Assistant (remember it may take up to a minute for the service to restart):
+### {% linkable_title Editing config via Samba/Windows Networking %}
 
-```yaml
-sensor:
-  - platform: random
-```
+Maybe you are not a big fan of our web editor and want to use a text editor on your computer instead. This is possible by sharing the configuration over the network using the Samba add-on, which can also be installed from the Hass.io add-on store. This will make your configuration accessible via the network tab on your computer.
 
-You'll know it worked when you see the new random sensor in your overview page. On the next page, we'll create an automation that uses this sensor to take an action.  
+Go to the add-on store and look for Samba in the core section. After you have installed the add-on, click on START. Hass.io should now be available in the networking tab on your computer.
```

Comment:
There's inconsistent formatting for "click on" things, some are

click on "THIS"

some are

click on "this"

some are

click on THIS

These really need to be the same, whatever is chosen

Reply:
I have adjusted it here, this was still old text. The text is capitalized because it's capitalized on the buttons too.